packages/sdk/schema/src/types/format.ts

```diff
@@ -26,16 +28,25 @@ const extend = (format: FormatEnum, type: TypeEnum, fields = {}) =>
     }),
   ).pipe(S.mutable);
 
+interface FormatSchemaCommon extends BaseProperty {
```

Comment:
can't this be imputed from the union?
e.g., if we add new format variants we have to manually create this mixin? (it's not "common" it's the "union").